ADOPTERS.md

```diff
@@ -1,5 +1,9 @@
 # Adopters
 
+Sysdig created Falco in 2016 to detect unexpected or suspicious activity using a rules engine on top of the data that comes from the sysdig kernel system call probe.
```

Comment:
This is an unnecessary detail. This file only purpose is here to list adopters, not to tell the story of the project or what it does.